src/platforms/dummy/dummyiaphandler.cpp

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "platforms/dummy/dummyiaphandler.h"
+#include "leakdetector.h"
+#include "logger.h"
+
+#include <QCoreApplication>
+
+namespace {
+Logger logger(LOG_IAP, "DummyIAPHandler");
```

Comment:
You are not using this. Remove it. And remove the logger.h... or use it somewhere in the methods.